modules/scripts/startup-script/files/install_docker.yml

```diff
@@ -0,0 +1,73 @@
+# Copyright 2024 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+- name: Install and configure Docker
+  hosts: all
+  become: true
+  vars:
+    enable_docker_world_writable: false
+  tasks:
+  - name: Check if docker is installed
+    ansible.builtin.stat:
+      path: /usr/bin/docker
+    register: docker_binary
+  - name: Download Docker Installer
+    ansible.builtin.get_url:
+      url: https://get.docker.com
+      dest: /tmp/get-docker.sh
+    when: not docker_binary.stat.exists
+  - name: Install Docker
+    ansible.builtin.command: sh /tmp/get-docker.sh
+    when: not docker_binary.stat.exists
+  - name: Create Docker socket override directory
+    ansible.builtin.file:
+      path: /etc/systemd/system/docker.socket.d
+      state: directory
+    when: enable_docker_world_writable
+  - name: Create Docker socket override configuration
+    ansible.builtin.copy:
+      dest: /etc/systemd/system/docker.socket.d/world-writable.conf
+      mode: 0644
+      content: |
+        [Socket]
+        SocketMode=0666
+    when: enable_docker_world_writable
+    notify:
+    - Reload SystemD
+    - Recreate Docker socket
+  - name: Delete Docker socket override configuration
```

Comment:
nit: readability for me is 2x better with whitespace between tasks.

Reply:
ack.

I have opposite problem. Whitespace signals to me the transition between tasks/handlers/etc. I think this is a question for setting yamllint or yamlfmt or ansible-lint to a setting and enforcing the standard repo wide.